api/tests/routes/public/collective/endpoints/get_collective_offers_test.py

```diff
@@ -42,14 +41,14 @@ def test_get_offers(self, client):
             {
                 "id": offer1.id,
                 "venueId": offer1.venueId,
-                "beginningDatetime": "2022-05-02T15:00:00",
+                "beginningDatetime": format_to_second_date(stock1.beginningDatetime),
```

Comment:
Je remets le commentaire que j'ai indiqué [ailleurs dans cette PR](https://github.com/pass-culture/pass-culture-main/pull/10066#discussion_r1445873613) et qui est peut-être passé à la trappe : 

> Est-ce qu'on a vraiment besoin de définir une fonction qui serait utilisée seulement dans les tests ? Est-ce que les tests ne pourraient pas utiliser la fonction de formatage de date utilisée par l'implémentation ?

En l'occurrence, il suffirait ici d'écrire : 

```suggestion
                "beginningDatetime": stock1.beginningDatetime.isoformat(timespec="seconds"),
```

C'est aussi lisible que `format_to_second_date()` et ça évite d'introduire une fonction d'une ligne qui n'est utilisée que dans les tests.